x/rollapp/keeper/msg_server_update_state.go

```diff
@@ -2,17 +2,23 @@ package keeper
 
 import (
 	"context"
+	"math"
 	"slices"
 
 	errorsmod "cosmossdk.io/errors"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 )
 
 func (k msgServer) UpdateState(goCtx context.Context, msg *types.MsgUpdateState) (*types.MsgUpdateStateResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	if msg.NumBlocks > math.MaxUint64-msg.StartHeight {
```

Comment:
u don;t prefer to put it in 
`func (msg *MsgUpdateState) ValidateBasic() error {` with all other validations?